server/start/initialize/index.ts

```diff
@@ -365,7 +357,8 @@ export function jobInitializeRun(context) {
         await getTemplateByName();
       }
     } catch (error) {
-      log('initialize:checkKibanaStatus', error.message || error);
+      if(error.name !== 'ResponseError')
+        log('initialize:checkKibanaStatus', error.message || error);
       context.wazuh.logger.error(error.message || error);
```

Comment:
The condition is only affecting the `log` function and no to Kibana logger. Should it be of this way?

Reply:
Thanks, if it could be interesting to try to control the kibana loggers